lms/djangoapps/shoppingcart/tests/test_views.py

```diff
@@ -918,7 +918,12 @@ class mock_get_order(object):  # pylint: disable=invalid-name
         """Mocks calls to EcommerceAPI.get_order. """
         patch = None
 
-        ORDER = copy.deepcopy(EcommerceApiTestMixin.ECOMMERCE_API_SUCCESSFUL_BODY)
+        ORDER = {
+            'status': OrderStatus.COMPLETE,
+            'number': EcommerceApiTestMixin.ORDER_NUMBER,
+            'payment_processor': 'cybersource',
+            'payment_parameters': {'orderNumber': EcommerceApiTestMixin.ORDER_NUMBER},
```

Comment:
This test data doesn't appear to be representative of the data returned from the ecommerce API; payment information is keyed under `payment_data`, separate from the information keyed under `order`.